src/main/kotlin/kscript/app/AppHelpers.kt

```diff
@@ -275,15 +318,15 @@ sourceSets.test.java.srcDirs 'src'
 
         // also symlink all includes
         includeURLs.distinctBy { it.fileName() }
-          .forEach {
+            .forEach {
```

Comment:
This came after auto-reformating code. I have my IDE set on tab size = indent = continuation = 4 spaces. Should I use something else ?